cps/templates/author.html

```diff
@@ -0,0 +1,65 @@
+{% extends "layout.html" %}
+{% block body %}
+<h2>{{title}}</h2>
+
+{% if author is not none %}
+<section class="author-bio">
+  {%if author['image_url'] is not none %}
+  <img src="{{author.image_url}}" alt="{{author.name}}" class="author-photo pull-left">
```

Comment:
When `author` attributes contains unicode characters (e.g. name="Honoré de Balzac") an exception occurs. 
I could fix it by replacing any occurrence of `{{author.name}}` with `{{author.name.decode('utf-8')}}`.

```
InsecurePlatformWarning
[2017-07-14 02:08:58,027] ERROR in app: Exception on /author/120 [GET]
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Library/Python/2.7/site-packages/flask_login/utils.py", line 228, in decorated_view
    return func(*args, **kwargs)
  File "/Users/Janek/Downloads/calibre/calibre-web/cps/web.py", line 1109, in author
    print author_info
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 5: ordinal not in range(128)
ERROR:tornado.access:500 GET /author/120 (::1) 2652.11ms
```